fv3core/stencils/remapping.py

```diff
@@ -55,6 +55,8 @@ def compute(
         pt.shape, grid.compute_origin(), cache_key="remapping_cvm"
     )
 
+    remap_part2_compute = remap_part2.Remapping_Part2()
```

Comment:
be sure to use the caching util until this is done in an object init method

Reply:
Done!